site/content/docs/5.2/forms/input-group.md

```diff
@@ -289,6 +325,12 @@ Input groups include support for custom selects and custom file inputs. Browser
 
 ### Custom file input
 
+{{< ods-incompatibility-alert >}}
+These form variants should not be used because they do not respect the Orange Design System specifications.
+
+Please refer to the [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) guidelines and to the [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.
```

Comment:
Same for Boosted file inputs ?

Reply:
Here, do you want to add a link towards Boosted Select page too? Here, unless I missed something, I don't think it's necessary to do that.